From 765eea0cdb14691583674fab7c04bddc7fd971ad Mon Sep 17 00:00:00 2001
From: root <root@devel.lab.contextream.local>
Date: Sun, 4 Sep 2016 18:00:12 +0300
Subject: [PATCH] add install option for Centos

---
 util/install.sh | 15 +++++++++++++--
 1 file changed, 13 insertions(+), 2 deletions(-)

diff --git a/util/install.sh b/util/install.sh
index 60eb7c1..a68119f 100755
--- a/util/install.sh
+++ b/util/install.sh
@@ -33,6 +33,17 @@ ARCH=`uname -m`
 if [ "$ARCH" = "x86_64" ]; then ARCH="amd64"; fi
 if [ "$ARCH" = "i686" ]; then ARCH="i386"; fi
 
+test -e /etc/centos-release && DIST="CentOS"
+if [ "$DIST" = "CentOS" ]; then
+    install='sudo yum -y install'
+    remove='sudo yum -y erase'
+    pkginst='sudo rpm -ivh'
+    # Prereqs for this script
+    if ! which lsb_release &> /dev/null; then
+        $install redhat-lsb-core
+    fi
+fi
+
 test -e /etc/debian_version && DIST="Debian"
 grep Ubuntu /etc/lsb-release &> /dev/null && DIST="Ubuntu"
 if [ "$DIST" = "Ubuntu" ] || [ "$DIST" = "Debian" ]; then
@@ -68,8 +79,8 @@ echo "Detected Linux distribution: $DIST $RELEASE $CODENAME $ARCH"
 KERNEL_NAME=`uname -r`
 KERNEL_HEADERS=kernel-headers-${KERNEL_NAME}
 
-if ! echo $DIST | egrep 'Ubuntu|Debian|Fedora|RedHatEnterpriseServer'; then
-    echo "Install.sh currently only supports Ubuntu, Debian, RedHat and Fedora."
+if ! echo $DIST | egrep 'Ubuntu|Debian|Fedora|RedHatEnterpriseServer|CentOS'; then
+    echo "Install.sh currently only supports Ubuntu, Debian, RedHat, CentOS and Fedora."
     exit 1
 fi
 
-- 
1.8.3.1

